tests/benchmark_test_inputs/AE/csv_elements/storage_1.csv

```diff
@@ -0,0 +1,15 @@
+,unit,storage capacity,input power,output power
```

Comment:
@smartie2076 - does it correspond to the 1MWh capacity battery described in the benchmark test A+E #106 ?

Reply:
Yes: Cell `installedCap`/`storage capacity` has to be defined as 1000.